src/OpenLoco/src/Objects/IndustryObject.h

```diff
@@ -67,6 +67,7 @@ namespace OpenLoco
     struct IndustryObject
     {
         static constexpr auto kObjectType = ObjectType::industry;
+        static constexpr auto cargoTypeNull = 0xFF;
```

Comment:
We prefix constants with a k. There are a lot of other places where we use 0xFf for null cargo as well. Not sure if having the constant here is best

Reply:
I think it is good to remove magic constants, even if widely used.
Perhaps an enum class solution like IndustryId would be a good alternative?

Reply:
I'm not saying don't do this I'm saying we should maybe put it in Object.h or something like that so it can be accessed by all the other headers that use cargo object references. Also you should use 0xFFU we don't want this to be signed really.

Reply:
I've moved the constant, but I'm doubting whether its name should remain specific to cargo types. It appears there are lot of objects that are compared to 0xFF to determine whether they're null but maybe that is something for a later refactoring.

Reply:
I think what is more important is not the value but the semantics of the value. Best to have one constant per semantically-different use of 0xFFU. Otherwise if you group together multiple semantics in one value, it becomes harder to decouple later if we need to.